charts/katib/templates/database/postgres-deployment.yaml

```diff
@@ -0,0 +1,88 @@
+{{- if and .Values.database.postgres.enabled (eq .Values.database.type "postgres") }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "katib.fullname" . }}-postgres
+  namespace: {{ include "katib.namespace" . }}
+  labels:
+    {{- include "katib.postgres.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "katib.postgres.selectorLabels" . | nindent 6 }}
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        {{- include "katib.postgres.selectorLabels" . | nindent 8 }}
+        sidecar.istio.io/inject: "false"
+      annotations:
+        {{- with .Values.database.postgres.annotations }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- with .Values.global.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.database.postgres.podSecurityContext }}
+      securityContext:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: katib-postgres
+          image: postgres:14.5-alpine
+          {{- with .Values.database.postgres.securityContext }}
+          securityContext:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          envFrom:
+            - secretRef:
+                name: {{ include "katib.database.secretName" . }}
+          env:
+            - name: PGDATA
+              value: /var/lib/postgresql/data/pgdata
+            {{- with .Values.database.postgres.env }}
+            {{- toYaml . | nindent 12 }}
+            {{- end }}
+          ports:
+            - name: postgres
+              containerPort: 5432
```

Comment:
my opinion is that ports should be configurable unless it's hard-coded somewhere in katib